### PR TITLE
Feature/package updates

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
Pull Request
**Are you a current Fivetran customer?** 

Internal
**What change(s) does this PR introduce?** 

We are applying standardization updates to be more consistent in our documentation.
**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [x] Yes (please provide breaking change details below.)
- [] No  (please provide an explanation as to how the change is non-breaking below.)
This won't be necessarily be breaking, but this will be a major change to the look and feel of our package.
**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [x] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [] Postgres
- [] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
📚 :

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 packages:
-- package: fivetran/lever_source
-  version: [">=0.3.0", "<0.4.0"]
+# - package: fivetran/lever_source
+#   version: [">=0.3.0", "<0.4.0"]
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:3]:
```suggestion
# - package: fivetran/lever_source
#   version: [">=0.3.0", "<0.4.0"]
  - git: https://github.com/fivetran/dbt_lever_source.git
    revision: feature/packages_updates
    warn-unpinned: false
```

File: README.md [hunk 2]
```diff
@@ -21,32 +35,31 @@ This package contains transformation models, designed to work simultaneously wit
...
+
+## Step 1: Prerequisites
+To use this dbt package, you must have the following:
 
-Include in your `packages.yml`
+- At least one Fivetran Lever connector syncing data into your destination.
+- A **BigQuery**, **Snowflake**, or **Redshift** destination.
 
+## Step 2: Install the package
+Include the following lever_source package version in your `packages.yml` file:
```
Comment (fivetran-joemarkiewicz) [inline README.md:47]:
```suggestion
Include the following lever package version in your `packages.yml` file:
```

File: README.md [hunk 3]
```diff
@@ -58,6 +71,9 @@ vars:
     lever_using_requisitions: false # Disable if you do not have the requisition table, or if you do not want requisition related metrics reported
     lever_using_posting_tag: false # disable if you do not have (or want) the postings tag table
 ```
+## (Optional) Step 6: Additional configurations
```
Comment (fivetran-joemarkiewicz) [inline README.md:74]:
```suggestion
## (Optional) Step 5: Additional configurations
```

File: README.md [hunk 4]
```diff
@@ -72,23 +88,61 @@ vars:
...
+
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_lever/blob/main/dbt_project.yml) variable declarations to see the expected names.
+
+```yml
+vars:
+    lever_<default_source_table_name>_identifier: your_table_name 
+```
+</details>
+
+## (Optional) Step 7: Orchestrate your models with Fivetran Transformations for dbt Core™
```
Comment (fivetran-joemarkiewicz) [inline README.md:113]:
```suggestion
## (Optional) Step 6: Orchestrate your models with Fivetran Transformations for dbt Core™
```

File: README.md [hunk 5]
```diff
@@ -72,23 +88,61 @@ vars:
...
+    lever_source:
+      +schema: my_new_schema_name # leave blank for just the target_schema
+    lever:
+      +schema: my_new_schema_name # leave blank for just the target_schema
+```
+    
+### Change the source table references
+If an individual source table has a different name than the package expects, add the table name as it appears in your destination to the respective variable:
+
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_lever/blob/main/dbt_project.yml) variable declarations to see the expected names.
```
Comment (fivetran-joemarkiewicz) [inline README.md:105]: This way it is easier to see what the source tables are directly from the source package.
```suggestion
> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_lever_source/blob/main/dbt_project.yml) variable declarations to see the expected names.
```

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-reneeli I know CircleCi is failing, but I think it is due to the packages.yml. I did a review and have a few suggestions that should be applied before merging.
- [ ] CHANGELOG suggested change
- [ ] A few README updates including updating the links to the final models
- [ ] Updating the packages.yml. I believe this is what is necessary to pass the tests. But be sure to switch before merging.

**Review by fivetran-avinash — COMMENTED** [review]
Hello @fivetran-catfritz! Everything looks good, just some minor tweaks to clean up the Readme.

One other thing--if you can update your integration_tests/requirements.txt files to the following specs below and include it in the PR. Joe mentioned that we should start updating requirements files going forward since the original 1.0.0 versions are starting to throw errors, just something to keep in mind!

Screen Shot 2022-09-26 at 2 31 50 PM

**Pull request merged**